code/game/gamemodes/revolution/revolution.dm

```diff
@@ -146,10 +146,12 @@
 		"backpack" = slot_in_backpack,
 		"left pocket" = slot_l_store,
 		"right pocket" = slot_r_store,
-		"left hand" = slot_l_hand,
-		"right hand" = slot_r_hand,
 	)
 	var/where = mob.equip_in_one_of_slots(T, slots)
+	if(!where)
```

Comment:
You repeat this several times. I'd rather equip_in_one_of_slots had the behaviour instead.